### DISCUSSION TITLE
How to use XYZ Plot for Pinpoint Adjust?

### DISCUSSION THREAD

**Discussion by ericnd01** [hako-mikan/sd-webui-supermerger#314] [Q&A]
I am trying to use the XYZ plot to test different values for brightness and color (COL1, COL2, COL3).  When choosing "Pinpoint Adjust" as the X Type, how do I properly enter the values?  If I enter something like "0,0,0,0,7,3,6,6", I get 8 iterations and no real difference.  The grid shows only one value per iteration.

**Comment by daileta1** [comment]
This too me forever to figure out, but here it is:
Set your X or Y to pinpoint adjust and list what you want to adjust. For example if its the first color, only COL1 would go there. Then, select alpha (just plain old alpha) and put the value there. If you have 0,2,4 there, it'll run through COL1 as 0, then 2, then 4. I don't know why you select alpha to make the value for adjust, but that's how it works.
